docs/development-guide.md

```diff
@@ -34,7 +34,15 @@ it is **highly** encouraged to:
 * Fork the [ceph-csi repo](https://github.com/ceph/ceph-csi) on Github.
 * Add your fork as a git remote:
    `$ git remote add fork https://github.com/<your-github-username>/ceph-csi`
-
+* Set up a pre-commit hook to catch issues locally.
+   `$ pip install pre-commit==2.5.1`
+   `$ pre-commit install`
+   See the [pre-commit installaton
```

Comment:
I do not think this renders as you intended. Please check https://github.com/ceph/ceph-csi/blob/9dcf1357b7643650aee73ea6ed9ae3d6ba754185/docs/development-guide.md for that.

You'll probably need to indent a little more?